src/hdf5_io.cpp

```diff
@@ -381,18 +381,11 @@ bool h5_read_string(char const* filename, char const* varname, std::string& outp
 {
   if (!fileExists(filename)) {
     std::cerr << ERROR_INFO << "File '" << filename << "' not found." << std::endl;
-    //TODO: Exception? Only error code?
+    //TODO: File not found - no idea what error code to use
```

Comment:
Why did you revert all changes in this file?

Reply:
No idea, it was not my intention, good catch. Somehow the old file must have snuck in.